src/main/cpp/Robot.cpp

```diff
@@ -46,12 +46,15 @@ void Robot::TeleopInit()
   // teleop starts running. If you want the autonomous to
   // continue until interrupted by another command, remove
   // this line or comment it out.
-
+  
   if (m_autonomousCommand)
   {
     m_autonomousCommand->Cancel();
     m_autonomousCommand.reset();
   }
+
+  //m_container.ZeroHeading();
```

Comment:
Why aren't we resetting the heading in teleop init?

Reply:
Corey said it wasn't working in teleop init. It can be investigated. I think the root issue is when we start auto, we know where we are facing so we can reset the heading specifically to be field oriented front. When auto is complete, we don't know which way we are facing and it messes up being able to get our heading perfectly aligned the way Ian wants it.